src/game/Miscellaneous/Language.h

```diff
@@ -1297,5 +1297,8 @@ enum TrinityStrings
     LANG_INSTANCE_BIND_MISMATCH                   = 11014,
     LANG_CREATURE_NOT_AI_ENABLED                  = 11015,
     LANG_SELECT_PLAYER_OR_PET                     = 11016,
+
+    // Mute for player 2 hour
+    LANG_MUTED_PLAYER   = 11016,
```

Comment:
can you change it into:

``LANG_MUTED_PLAYER   = 11017, // Mute for player 2 hour``

in order to correct the number (11017 instead of 11016 which is already used above)

and also it will be only 1 line more instead of 3, otherwise this file will get more and more complex as soon as we add more constants